bl/parser.js

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
+
+'use strict';
+const Parser = require('../.lib/parser.js');
+
+/**
+ * Recognizes the accesses to the platform Booklist
+ * @param  {Object} parsedUrl an object representing the URL to analyze
+ *                            main attributes: pathname, query, hostname
+ * @param  {Object} ec        an object representing the EC whose URL is being analyzed
+ * @return {Object} the result
+ */
+module.exports = new Parser(function analyseEC(parsedUrl, ec) {
+  let result = {};
+  let path   = parsedUrl.pathname;
+  // uncomment this line if you need parameters
+  let param = parsedUrl.query || {};
+
+  // use console.error for debuging
+  // console.error(parsedUrl);
+
+  let match;
+
+  if (/^\/SearchResults.aspx$/i.test(path)) {
+    // https://www.booklistonline.com/SearchResults.aspx
+    result.rtype    = 'SEARCH';
+    result.mime     = 'HTML';
+  } else if ((match = /^\/([a-zA-Z0-9-]+)\/pid=([0-9]+)$/i.exec(path)) !== null) {
```

Comment:
This parser is to permissive, with this parser, you will count 
- https://www.booklistonline.com/faq
- https://www.booklistonline.com/booklist-blog
- https://www.booklistonline.com/newsletters
as RECORD HTML.
Maybe the solution is to include the presence of the param PID

Reply:
Hi @felixleo22 
Why would this parser count https://www.booklistonline.com/faq ?
I did include /pid=([0-9]+) in regex. It would not count the url you listed above since they don't contain "pid"

Reply:
oh my bad